### ISSUE TITLE
Add windows configuration files for JavaFX

### ISSUE THREAD

**Issue by tiainen** [gluonhq/substrate#205]
Define the jni and reflection configuration files for the windows target when building a JavaFX application. Also specify the required JavaFX native libraries that should be linked into the final executable.

**Comment by iw-an** [comment]
Hi, is there an ETA on this?

**Comment by warmuuh** [comment]
i tried working around the first issue by injecting some dummy config files into the jar, just for testing, but hit the second wall of the plugin trying to download the static libs for the platform.
for reference:

```
[ERROR] Failed to execute goal com.gluonhq:client-maven-plugin:0.1.15:link (default-cli) on project milkman-dist-graal: Error: Error downloading zips: Error downloading from http://download2.gluonhq.com/substrate/staticjdk/labs-staticjdk-windows-x86_64-gvm-14-ea+5.zip 
into C:\Users\peter\.gluon\substrate\labs-staticjdk-windows-x86_64-gvm-14-ea+5.zip: http://download2.gluonhq.com/substrate/staticjdk/labs-staticjdk-windows-x86_64-gvm-14-ea+5.zip,
```

**Comment by tiainen** [comment]
Currently blocked on https://github.com/oracle/graal/issues/2406

**Comment by tsatatwer** [comment]
oracle/graal#2406 is closed ......so please any update ???....thanks

**Comment by tiainen** [comment]
Fixed with #581
